### DISCUSSION TITLE
Can I use plane as helpdesk?

### DISCUSSION THREAD

**Discussion by link89** [makeplane/plane#6197] [Q&A] [answered]
Plane looks to be a cool Jira alternative. Can I build a helpdesk with plane? I hope that employees can create tickets when they need help. And the ticket will be assigned to dedicated supporter accordingly.

**Comment by danciaclara** [comment, marked as answer]
Plane isn’t specifically designed as a helpdesk solution, but you can adapt it for that purpose. You can set up a project where employees create tickets, and then use features like assignees and statuses to manage them. Intake Forms (coming soon on the Business plan) can help streamline support ticket creation. External users can submit tickets easily via a shared form link, making it even more efficient to manage helpdesk requests in Plane.
